autodistill/classification/__init__.py

```diff
@@ -1,4 +1,4 @@
-from autodistill.classification.classification_base_model import ClassificationBaseModel
-from autodistill.classification.classification_target_model import (
-    ClassificationTargetModel,
-)
+from autodistill.classification.classification_base_model import \
+    ClassificationBaseModel
```

Comment:
Why'd this break to multiple lines like this?

Reply:
`black`, which we are using for linting, enforces this when a line gets sufficiently long.